shared/clinical-code-sets/medications/serotonin-norepinephrine-reuptake-inhibitors/1/README.md

```diff
@@ -0,0 +1,18 @@
+# Serotonin and Norepinephrine Reuptake Inhibitors (SNRI) 
+
+This code set was created from getset.ga, using the following list from the PI of RQ051:
+-- Desvenlafaxine
+-- Duloxetine
+-- Duloxetine hydrochloride
+-- Venlafaxine hydrochloride
```

Comment:
Can't see and codes for this in the code files. I think you need to search for just "venlafaxine" rather than "venlafaxine hydrochloride"